pkg/client/cache/eventqueue.go

```diff
@@ -355,6 +355,30 @@ func (eq *EventQueue) ListConsumed() bool {
 	return eq.lastReplaceKey == ""
 }
 
+// Resync will touch all objects to put them into the processing queue
```

Comment:
@ironcladlou @pmorie @kargakis someone needs to review this (it complies but haven't tested) based on the latest changes to Pop and Resync upstream.  I don't remember the exact semantics of the event queue but want a sanity check.